build.fsx

```diff
@@ -87,6 +88,25 @@ Target "LibraryNuGet" (fun _ ->
 // Build a NuGet package
 Target "TemplatesNuGet" (fun _ ->
 
+    // turn on macOS
```

Comment:
VS for Mac add-ins require the nuget package to be an add-in file in the project, so this nuget needs to have a consistent name. This copies the versioned packed to the VS for Mac extension folder and removes the version number,